packages/events/src/EventBoundary.ts

```diff
@@ -497,9 +497,10 @@ export class EventBoundary
 
         if (displayObject._mask)
         {
-            const mask = displayObject._mask as any;
+            const maskObject = ((displayObject._mask as any).isMaskData
+                ? (displayObject._mask as any).maskObject : displayObject._mask);
```

Comment:
The double-any cast here is kinda ugly. Can we either use real types rather than making this worse?

Reply:
Why not check for `instanceof MaskData` or `'isMaskData` in displayObject._mask`? Trying to avoid the casting.

Reply:
We have similarly ugly mask data checks elsewhere already. Maybe we should add the getter `DisplayObject#maskObject`? Or just a helper method to extract the mask object.

Reply:
I think a normalized, non public getter (eg maskObject) sounds like a good idea. But if there are more places than this, another PR is probably better for that cleanup bit.